### ISSUE TITLE
Warning and error in install-vacation-request-sampleapp script related to spring security core plugin (Grails 2.0.0)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeewan2015/grails-activiti-plugin#23]
```
Please see 
http://grails.1312388.n4.nabble.com/Warning-and-error-in-gant-script-that-instal
l-spring-security-core-plugin-td4236367.html

```

Original issue reported on code.google.com by `limchee...@vobject.com` on 27 Dec 2011 at 12:00

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 3fce963e5e9e.
```

Original comment by `limchee...@vobject.com` on 30 Dec 2011 at 3:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Re-open due to issues stated at 
http://grails.1312388.n4.nabble.com/Warning-and-error-in-gant-script-that-instal
l-spring-security-core-plugin-td4236367.html#a4261029
```

Original comment by `limchee...@vobject.com` on 4 Jan 2012 at 12:27
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
fixed in 5.9 released.
```

Original comment by `limchee...@vobject.com` on 11 Jul 2012 at 3:57
- Changed state: **Fixed**
